.github/workflows/publish-dev.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: PublishDev
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - name: Install Dependencies with NPM
+        id: installation
+        run: |
+          npm ci --no-optional
+      - name: Publish Package to NPM
```

Comment:
this file and the other publish yaml can be merged if we use a conditional like this:
```
      - name: Publish Beta Package to NPM
        id: publishing-beta
        run: npm publish --access public --tag beta
        env:
          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
      - name: Publish Latest Package to NPM
        id: publishing-latest
        if: startsWith(github.ref, 'refs/tags/v')
        run: npm publish --access public --tag latest
        env:
          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```